### DISCUSSION TITLE
Redirect the default /welcome context using nginx

### DISCUSSION THREAD

**Discussion by Dnhatsave** [ONLYOFFICE/DocumentServer#1695] [Q&A]
Helo everyone,

I have a address were a like to host the onlyoffice eg: `https://www.example/url `.

I look that by default the onlyoffice comes with /`welcome `context eg: `http:my-ip:my-port/welcome` 
I want to know how can i **redirect** or **rewrite**  the default (/welcome) context to work at my environment like that: `https://www.example/url/welcome `

I've tried to use rewrite settings on nginx:
```
location /welcome/ {
    rewrite ^(.*)$ http://172.31.4.114:28096/colab/welcome/ redirect;
  }
```
But it gives me 404 Error not found as can you see at screenshoot 
![Screenshot from 2022-03-21 12-14-21](https://user-images.githubusercontent.com/43370602/159241905-c6f8239c-00f2-4173-b7e0-fbca9a433f4e.png)

**Comment by ShockwaveNN** [comment]
I'm not 100% sure because I'm not good with all nginx setup, but I think you need reverse proxy for that

I think all other parts of DocumentServer may be related to that part and if you move only `/welcome` page - all other URLs that DocumentServer uses will be confused

I'll move this to DocumentServer repo and convert to discussion, since it's not an issue per-se
Maybe someone will be able to help with exact configs

**Comment by Dnhatsave** [comment]
Thanks for replaying @ShockwaveNN ,

I like to know where i can find welcome.html file, what is the path ??
I've do proxy reverse as you suggest but now i want to change the `/welcome `context for another one.

**Reply by ShockwaveNN** [reply]
It's located at `/var/www/onlyoffice/documentserver-example/welcome/`
